bower.json

```diff
@@ -9,11 +9,13 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
+    "baguettebox.js": "~1.8",
     "bootstrap": "~3.3.7",
+    "html5shiv": "^3.7.3",
     "jquery": "<2.0.0",
     "jquery-colorbox": "~1",
-    "moment": "~2",
+    "katex": "^0.7.1",
```

Comment:
KaTeX is a very optional feature. And we can’t make users download all the node crap themselves just to get bower going. Or yarn and webpack, because bower is now deprecated. Or whatever they come up with in 6 months.